mods/ra/rules/structures.yaml

```diff
@@ -338,6 +340,21 @@ PDOX:
 		Duration: 20
 		KillCargo: yes
 		DisplayRadarPing: True
+	ChronoshiftPower@advancedchronoshift:
+		OrderName: AdvancedChronoshift
+		Prerequisites: structures.germany
```

Comment:
Killing or unpacking a german player's last MCV will cause their chronoshift to revert to the regular type and reset the timer.  You probably want to use a specific race prerequisite that is granted by the player actor.